web/admin/src/App.js

```diff
@@ -241,6 +263,14 @@ class App extends PureComponent {
                 {isPublished ? 'Live' : 'Draft'}
               </p>
             </span>
+            <button
```

Comment:
This UX feels clunky.  Button --> alert --> select/copy.  Instead of the button, we can just render the DD URL immediately.  The user can just select and copy without having to click a button and view a separate alert.